Only in wxWindows.orig: .cvsignore
Binary files wxWindows.orig/Debug/vc60.pdb and wxWindows/Debug/vc60.pdb differ
Binary files wxWindows.orig/lib/wxmswd.lib and wxWindows/lib/wxmswd.lib differ
Binary files wxWindows.orig/src/common/Debug/imagbmp.obj and wxWindows/src/common/Debug/imagbmp.obj differ
diff -rbu2 wxWindows.orig/src/common/imagbmp.cpp wxWindows/src/common/imagbmp.cpp
--- wxWindows.orig/src/common/imagbmp.cpp	Tue Jul 02 09:25:44 2002
+++ wxWindows/src/common/imagbmp.cpp	Tue Aug 13 15:21:00 2002
@@ -1242,7 +1242,7 @@
     static const char *icotxt  = "icon";
 
-    wxInt32 *riff32 = (wxInt32 *) rifftxt;
-    wxInt32 *list32 = (wxInt32 *) listtxt;
-    wxInt32 *ico32  = (wxInt32 *) icotxt;
+    wxInt32 riff32 = (wxInt32) rifftxt;
+    wxInt32 list32 = (wxInt32) listtxt;
+    wxInt32 ico32  = (wxInt32) icotxt;
 
     int iIcon = 0;
@@ -1250,5 +1250,5 @@
     stream.SeekI(0);
     stream.Read(&FCC1, 4);
-    if ( FCC1 != *riff32 )
+    if ( FCC1 != riff32 )
         return FALSE;
 
@@ -1262,5 +1262,5 @@
         if (datalen % 2 == 1) datalen ++ ;
         //now either data or a FCC
-        if ( (FCC1 == *riff32) || (FCC1 == *list32) )
+        if ( (FCC1 == riff32) || (FCC1 == list32) )
         {
             stream.Read(&FCC2, 4);
@@ -1268,5 +1268,5 @@
         else
         {
-            if (FCC1 == *ico32 && iIcon >= index)
+            if (FCC1 == ico32 && iIcon >= index)
             {
                 return DoLoadFile(image, stream, verbose, -1);
@@ -1275,5 +1275,5 @@
             {
                 stream.SeekI(stream.TellI() + datalen);
-                if ( FCC1 == *ico32 )
+                if ( FCC1 == ico32 )
                     iIcon ++;
             }
@@ -1294,7 +1294,7 @@
     static const char *anihtxt = "anih";
 
-    wxInt32 *riff32 = (wxInt32 *) rifftxt;
-    wxInt32 *list32 = (wxInt32 *) listtxt;
-    wxInt32 *anih32 = (wxInt32 *) anihtxt;
+    wxInt32 riff32 = (wxInt32) rifftxt;
+    wxInt32 list32 = (wxInt32) listtxt;
+    wxInt32 anih32 = (wxInt32) anihtxt;
 
     stream.SeekI(0);
@@ -1302,5 +1302,5 @@
         return FALSE;
 
-    if ( FCC1 != *riff32 )
+    if ( FCC1 != riff32 )
         return FALSE;
 
@@ -1308,5 +1308,5 @@
     while ( stream.IsOk() )
     {
-        if ( FCC1 == *anih32 )
+        if ( FCC1 == anih32 )
             return TRUE;
         // we always have a data size:
@@ -1316,5 +1316,5 @@
         if (datalen % 2 == 1) datalen ++ ;
         // now either data or a FCC:
-        if ( (FCC1 == *riff32) || (FCC1 == *list32) )
+        if ( (FCC1 == riff32) || (FCC1 == list32) )
         {
             stream.Read(&FCC2, 4);
@@ -1344,11 +1344,11 @@
     static const char *anihtxt = "anih";
 
-    wxInt32 *riff32 = (wxInt32 *) rifftxt;
-    wxInt32 *list32 = (wxInt32 *) listtxt;
-    wxInt32 *anih32 = (wxInt32 *) anihtxt;
+    wxInt32 riff32 = (wxInt32) rifftxt;
+    wxInt32 list32 = (wxInt32) listtxt;
+    wxInt32 anih32 = (wxInt32) anihtxt;
 
     stream.SeekI(0);
     stream.Read(&FCC1, 4);
-    if ( FCC1 != *riff32 )
+    if ( FCC1 != riff32 )
         return wxNOT_FOUND;
 
@@ -1362,5 +1362,5 @@
         if (datalen % 2 == 1) datalen ++ ;
         // now either data or a FCC:
-        if ( (FCC1 == *riff32) || (FCC1 == *list32) )
+        if ( (FCC1 == riff32) || (FCC1 == list32) )
         {
             stream.Read(&FCC2, 4);
@@ -1368,5 +1368,5 @@
         else
         {
-            if ( FCC1 == *anih32 )
+            if ( FCC1 == anih32 )
             {
                 wxUint32 *pData = new wxUint32[datalen/4];

 	  	 
